llvm/test/Transforms/VectorCombine/AArch64/sve-interleave-splat.ll

```diff
@@ -0,0 +1,10 @@
+; RUN: opt -mtriple=aarch64-unknown-linux-gnu -mattr=+sve -passes=vector-combine %s -S -o - | FileCheck %s
```

Comment:
I think you misunderstood my previous comment. By adding `-mattr=+sve` here this no-longer tests the crash you've fixed (as the crash in `BasicTTIImpl` is only possible without `+sve`).